compiler/resolution/wrappers.cpp

```diff
@@ -483,36 +437,6 @@ static void handleDefaultArg(FnSymbol *fn, CallExpr* call,
   body->flattenAndRemove();
 }
 
```

Comment:
Please add a note to the PR message about what allowed you to remove addDefaultTokensAndReorder(), replaceDefaultTokensWithDefaults(), fixDefaultArgumentsInWrapCall()